Cython/Utility/Optimize.c

```diff
@@ -609,9 +610,12 @@ static double __Pyx__PyObject_AsDouble(PyObject* obj) {
     if (likely(nb) && likely(nb->nb_float)) {
         float_value = nb->nb_float(obj);
         if (likely(float_value) && unlikely(!PyFloat_Check(float_value))) {
+            __Pyx_TypeName float_value_type_name =
+                __Pyx_PyType_GetName(Py_TYPE(float_value));
```

Comment:
```suggestion
            __Pyx_TypeName float_value_type_name = __Pyx_PyType_GetName(Py_TYPE(float_value));
```

Reply:
(I previously worked in projects with a strict 80 column limit and my usual setup is putting 4-5 terminals with 80 column next to each other; so I have a tendency to break lines. Anyway I will of course try to stay with the project preferences and do less wrapping here.